download_from_csv.py

```diff
@@ -86,20 +188,15 @@ def download_finish(d):
         print '[\033[93mSkipping\033[00m] %s by %s' % \
             (song['name'], song['artist'])
         continue
-    opts = {
-        'format': 'bestaudio/best',
-        'forcejson': True,
-        'postprocessors': [{
-            'key': 'FFmpegExtractAudio',
-            'preferredcodec': 'mp3',
-            'preferredquality': '256',
-        }],
-        'progress_hooks': [download_finish],
-        'logger': MyLogger(),
-        'outtmpl': folder + '/' + song['name'] + ' - ' + song['artist'] + '.%(ext)s'
-    }
-    url = ' '.join([song['name'], song['artist'], 'audio', 'youtube'])
-    url = 'gvsearch1:' + url
+    opts['progress_hooks'] = [download_finish]
+    opts['logger'] = MyLogger()
+    opts['outtmpl'] = folder + '/' + song['name']\
+        + ' - ' + song['artist'] + '.%(ext)s'
+    url = ' '.join([song['name'], song['artist']])
+    if args.search:
```

Comment:
1. No space will be there between `url` and `args.search`
2. In case the user chooses not to provide `args.search`, append 'audio' to the current `url`
3. Instead of adding strings, use `'ytsearch: %s %s' % (url, 'audio')`, as it significantly more faster than adding strings (as strings in python are immutable)